src/modules/endpoint/models/CategoryModel.js

```diff
@@ -1,53 +1,77 @@
+// @flow
+
+type categoryModel = {
+  id: number,
+  url: string,
+  title: string,
+  parentId: number,
+  parentUrl: string,
+  content: string,
+  thumbnail: string,
+  order: number,
+  availableLanguages: any
+}
+
 class CategoryModel {
```

Comment:
Do we actually need these classes? I think type definition was the sole purpose of them... 🤔

Reply:
nope, good suggestion